### ISSUE TITLE
"No connection to ownCloud" error message contains redundant sentence

### ISSUE THREAD

**Issue by phil-davis** [owncloud/client#3794]
Seeing https://github.com/owncloud/client/issues/3793 made me look at this. I turned off my network connection, pause/resumed sync and waited for it to give the connection error:
![noconnection](https://cloud.githubusercontent.com/assets/1535615/9766687/05a03714-573a-11e5-82b1-0d1ede8ba2e0.png)
"No connection" and "Unable to connect" seem to say the same thing twice.
Host x.y not found seems a reasonable more-detailed reason for "No connection", but given that the host name has already been mentioned (twice), it seems a waste of screen space to mention it again.
Maybe the building of the error message text here can be optimized.

**Comment by Dianafg76** [comment]
I tested this issue, and now it is working OK
Desktop v ownCloud-2.0.2.2713-nightly20150914.pkg
Closed

**Comment by guruz** [comment]
@Dianafg76 How did you test it?
I just noticed I still get redundancy if the server throws a message like ```Bad Gateway```

**Comment by Dianafg76** [comment]
@guruz I tested again, and when I turned off my network connection and waited for it to give the connection error, the text is not repeated:
![screen shot 2015-09-16 at 09 20 00](https://cloud.githubusercontent.com/assets/12032058/9898717/ddb33394-5c54-11e5-8dff-1cfedb1de61d.png)

**Comment by guruz** [comment]
@Dianafg76 Yes, that error is caused by another codepath, one layer earlier (when it tries to establish the connection).. :)
I'll look into mine for 2.1
